app/views/helpers/DeclarationHolderHelper.scala

```diff
@@ -104,25 +103,15 @@ class DeclarationHolderHelper @Inject()(
       case _                                                        => None
     }
 
-  private def bodyForClearanceArrived1040(implicit messages: Messages): List[String] =
-    content("body.clearance.eidr.1040") ++ content("body.exrr.roro.exports")
-
-  private def bodyForClearanceArrivedOthers(implicit messages: Messages): List[String] =
-    content("body.clearance.eidr.others") ++ content("body.exrr.roro.exports")
-
-  private def bodyForSimplifiedArrived(implicit messages: Messages): List[String] =
-    content("body.simplified") ++ content("body.exrr.roro.exports")
-
-  private def bodyWithLinkFor1007(appConfig: AppConfig, key: String)(implicit messages: Messages): List[String] =
-    List(
-      messages(
-        s"declaration.declarationHolder.body.$key.1007",
-        link(messages("declaration.declarationHolder.body.1007.link"), Call("GET", appConfig.permanentExportOrDispatch.section), "_blank")
-      )
-    )
+  def textForEoriRadiosWhenEXRR(implicit messages: Messages, request: JourneyRequest[_]): Html = {
+    val model = request.cacheModel
+    model.additionalDeclarationType match {
+      case Some(STANDARD_FRONTIER) | Some(SIMPLIFIED_FRONTIER) | Some(OCCASIONAL_FRONTIER) | Some(CLEARANCE_FRONTIER) =>
+        textForEoriRadiosWhenEXRR(model.parties)
```

Comment:
Bit confusing having same name for this function as the function calling it. Maybe rename the calling function `contentForEoriRadiosWhenEXRR` just to differentiate the two.